kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousConfigUtils.java

```diff
@@ -32,15 +35,33 @@ public class KroxyliciousConfigUtils {
      * @return builder
      */
     public static ConfigurationBuilder proxy(String clusterBootstrapServers) {
-        return new ConfigurationBuilder().addToVirtualClusters(DEFAULT_VIRTUAL_CLUSTER, new VirtualClusterBuilder()
-                .withNewTargetCluster()
-                .withBootstrapServers(clusterBootstrapServers)
-                .endTargetCluster()
-                .withClusterNetworkAddressConfigProvider(
-                        new ClusterNetworkAddressConfigProviderDefinitionBuilder(PortPerBrokerClusterNetworkAddressConfigProvider.class.getName())
-                                .withConfig("bootstrapAddress", DEFAULT_PROXY_BOOTSTRAP)
-                                .build())
-                .build());
+        return proxy(clusterBootstrapServers, DEFAULT_VIRTUAL_CLUSTER);
+    }
+
+    /**
+     * Create a KroxyliciousConfigBuilder with a single virtual cluster configured to
```

Comment:
there's now the potential for >1 virtual clusters.

Reply:
Thanks for catching that. fixed with 89c35e8be4ee8646a3af4fc8a952d1ba4a8e29c4